lib/draft/schema.json

```diff
@@ -793,6 +793,7 @@
               "moisture",
               "ph",
               "pir",
+              "pressure",
```

Comment:
This needs to be added to https://github.com/lorawan-schema/draft-devices

Reply:
I just did that, please rebase your changeset against `master`

Reply:
The changeset has been rebased against `master`.

Just a side note:
- does it make sense to convert this to a git submodule?
- there's a tab character in front of `pressure` that can be fixed next time.

Reply:
> * does it make sense to convert this to a git submodule?

It is in fact a git subtree

> * there's a tab character in front of `pressure` that can be fixed next time.

Will fix next time, thanks